product_docs/docs/migration_toolkit/55/04_functionality_overview.mdx

```diff
@@ -29,6 +29,11 @@ Migration Toolkit also allows you to migrate database objects and data to an EDB
 -   Use the `-dropSchema` option to drop the existing schema and create a new schema prior to migration.
 -   On EDB Postgres Advanced Server, use the `-allDBLinks` option to migrate all Oracle database links.
 -   On EDB Postgres Advanced Server, use the `-copyViaDBLinkOra` option to enable the `dblink_ora` module.
+-   Use the `-connRetryCount <connection_attempts>` option to specify the number of retry attempts to perform. 
+-   Use the `-connRetryInterval <seconds>` option to specify the seconds to wait before each reconnection attempt.
+-   Use the `-abortConnOnFailure <true/false>` option to specify if to abort the migration when all reconnection attempts failed.
+    The default is `true`, which aborts the session if the connection fails after the the specified `-connRetryCount` threshold.
+-   Use the `-pgIdleTxSessionTimeOut <seconds>` to specify the PostgreSQL or EDB Postgres Advanced Server `idle_in_transaction_session_timeout`.
```

Comment:
I would suggest to rephrase "to specify the PostgreSQL or EDB Postgres Advanced Server idle_in_transaction_session_timeout"
AS
"to override the value of the PostgreSQL or EDB Postgres Advanced Server idle_in_transaction_session_timeout configuration option in the MTK connection session."